src/components/TodoItem.tsx

```diff
@@ -0,0 +1,85 @@
+import { ItoDo } from "../components/types/data";
+import { useState, useEffect, useRef } from "react";
+import React from "react";
+
+interface ItoDoItem extends ItoDo {
+  items: ItoDo[];
+  setItems: React.Dispatch<React.SetStateAction<ItoDo[]>>;
+  //editTodo: (id: number) => void;
+
+  removeTodo: (id: number) => void;
+  inputText: (id: number, title: string) => void;
+  toggleTodo: (id: number, title: string) => void;
+}
+
+const TodoItem: React.FC<ItoDoItem> = (props) => {
```

Comment:
Please use spread operator for props here

Reply:
done